tests/resources/notebook-controller/notebooks/generic-data-science-oauth-notebook.yaml

```
@@ -1,16 +1,15 @@
----
 apiVersion: kubeflow.org/v1
 kind: Notebook
 metadata:
-  name: thoth-minimal-oauth-notebook
+  name: jupyter-datascience-ubi8-python-3-8
   annotations:
     notebooks.opendatahub.io/inject-oauth: "true"
 spec:
   template:
     spec:
       containers:
-        - name: thoth-minimal-oauth-notebook
-          image: image-registry.openshift-image-registry.svc:5000/opendatahub/s2i-minimal-notebook:v0.3.0-py38
+        - name: jupyter-datascience-ubi8-python-3-8
+          image: qquay.io/opendatahub/notebooks:jupyter-datascience-ubi8-python-3.8
```

We want to test the images imported by the previous image streams, not those in quay.io. For example:

```suggestion
          image: image-registry.openshift-image-registry.svc:5000/opendatahub/jupyter-datascience-notebook:py3.8-v1
```

Review the other notebooks too.